docs/usage/.pages

```diff
@@ -9,6 +9,7 @@ nav:
       - 'Self-hosted': 'self-hosted-configuration.md'
       - 'Presets': 'config-presets.md'
       - 'Validation': 'config-validation.md'
+  - ... | mend-hosted
```

Comment:
Please use the `Closes` keyword [^1] in your top post. That way, GitHub automatically closes the issue when the PR is merged.

Here's the old text, and the new text that you should use:

```diff
- Addresses #30711
+ Closes #30711
```

[^1]: [GitHub Docs, Using keywords in issues and pull requests](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)

Reply:
Ah! Thanks for the info. Will do in future.

Reply:
Updated as suggested. Thanks